doc/man3/err_clear_last_constant_time.pod

```diff
@@ -0,0 +1,32 @@
+=pod
+
+=head1 NAME
+
+err_clear_last_constant_time

```

Comment:
This is not part of our public interface, and should therefore be located in `doc/internal/man3`